Stopping animation when prompting user and uninstall before install

fernride-jissing opened the pull request:
I found it useful to indicate to the user that the PC is waiting for them by stopping the animation while waiting for the user to enter something (for instance the disk password).

I also added an uninstall to the script when it finds the folder already installed - it might be a bit overkill, but I'm curious what you think about it.

nesono committed:
Stopping animation when prompting user and uninstall before install

emanuele-scarsella commented:
Sorry for the late response, thank you for this contribution, I agree that a visual indication that a user action is required would improve the user experience, I can totally see some poor user not seeing the prompt and uselessly waiting for a while before proceeding, and stopping the animation surely is a quick and easy way to convey that information...

although, I think a better solution might be to temporary replace the central logo with a lock (for password)  or a question mark (for other prompts) with some fade-zoom animation of sort... so I might implement that some day, but for the time being this solution will do the trick and the commit looks fine to me, good job :-) 

Before merging, please add your name to the [Contributions section](https://github.com/emanuele-scarsella/vortex-ubuntu-plymouth-theme?tab=readme-ov-file#handshake-contributions) of the [README.md](https://github.com/emanuele-scarsella/vortex-ubuntu-plymouth-theme/blob/main/README.md) file followed by a brief description of what you did so that people knows who to thank for that change, then I'll proceed with the merge :-)

Thank you again for taking the time to improve this theme, I really appreciate that ♡

fernride-jissing committed:
Replace tabs with spaces to align with rest of the file

fernride-jissing committed:
Delete README.txt

fernride-jissing committed:
Add me as contributor with short description

fernride-jissing commented:
@emanuele-scarsella thanks for the kind words! I really was happy to find your splash screen since it's coding the progress bar and doesn't require tons of images for handling the animations.

I removed the `README.txt` - not sure where this came from - and added myself to the contributor list with a short explanation of what I did.

I hope this finds you well.
